cutil/parser/httpfileresource/httpresource.go

```diff
@@ -0,0 +1,68 @@
+// Copyright © 2017 The Kubicorn Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package httpresource
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"regexp"
+)
+
+// HTTPResource could hold explicit information for HTTP calls
+// right now just helps us for type enforcment
+type HTTPResource struct{}
+
+// NewHTTPResource creates a new file resource fetcher
+func NewHTTPResource(fr HTTPResource) *HTTPResource {
```

Comment:
Well, my idea was that those empty structs where just place holders, so if in the future something was put in there, the usage would not change that much.